src/scss/index.scss

```diff
@@ -199,10 +199,6 @@ td:first-child {
 
 /* -------------------------------------------------------------------------- */
 
-.container-fluid {
-  max-width: 1300px;
-}
-
```

Comment:
I think this one can still be kept? (to preserve the max-width introduced earlier)

Reply:
on wide screens, that will center the content in the middle, with the top navbar spanning the full width:

![image](https://user-images.githubusercontent.com/1839645/76797957-df23c580-678b-11ea-933f-72a242f5e608.png)

Reply:
Ah, should there be some left alignment added then (until the centering is done)?

If that gets too complicated, fine with reverting to full width for now (since the wrapping bug is annoying enough)

Reply:
I can manually remove the left margin by over-riding the bootstrap CSS, though I feel like that's a not-good practice we have learned the hard way already 😆 ...let me know what you want me to do

Reply:
@choldgraf how come you have to wait for 25 seconds each time? Are you using the webpack `buid:dev` flow?

Reply:
Oh yikes, I just read https://github.com/pandas-dev/pydata-bootstrap-sphinx-theme/issues/127. I will look into this today!

Reply:
OK, let's then revert the max-width here. At least that fixes the main bug, and then @hoetmaaiers can work on a better solution

Reply:
Can you explain what you mean? Do you want `max-width` or not? Sorry, just can't tell from the language :-)

Reply:
Sorry, I meant "revert" as revert the max-width that is in master, so keeping the revert (removing the max-width) you already did here (and thus not "revert your revert" ;-))